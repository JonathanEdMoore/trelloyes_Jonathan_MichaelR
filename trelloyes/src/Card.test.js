import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';


it('renders without crashing', () =>{
  const div = document.createElement('div');
  ReactDOM.render(<Card />, div);
  ReactDOM.unmountComponentAtNode(div);
})

describe('<Card />', () => {
  const snap = renderer.create(<Card />).toJSON();
  expect(snap).toMatchSnapshot();
})
