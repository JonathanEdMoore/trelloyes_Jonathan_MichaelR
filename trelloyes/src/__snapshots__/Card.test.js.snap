// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    Delete
  </button>
  <h3 />
  <p />
</div>
`;
